drivers/i2c/i2c_mchp_xec.c

```diff
@@ -609,20 +982,38 @@ static int i2c_xec_slave_unregister(const struct device *dev,
 static const struct i2c_driver_api i2c_xec_driver_api = {
 	.configure = i2c_xec_configure,
 	.transfer = i2c_xec_transfer,
+	.recover_bus = i2c_xec_recover_bus,
 #ifdef CONFIG_I2C_SLAVE
-	.slave_register = i2c_xec_slave_register,
-	.slave_unregister = i2c_xec_slave_unregister,
+	.slave_register = i2c_xec_target_register,
+	.slave_unregister = i2c_xec_target_unregister,
 #endif
 };
 
 static int i2c_xec_init(const struct device *dev)
 {
+	const struct i2c_xec_config *cfg = dev->config;
 	struct i2c_xec_data *data =
 		(struct i2c_xec_data *const) (dev->data);
 	int ret;
 
+	ITM_send_str(0, "INIT ");
+
+	data->target_cfg = NULL;
+	data->started = 0;
 	data->pending_stop = 0;
-	data->slave_attached = false;
+	data->target_attached = false;
+
+	data->sda_gpio = device_get_binding(cfg->sda_gpio_label);
```

Comment:
@galak is the other comment below  just a naming convention? I could not find any guidelines against this binding here. 
>>> This should be pinctrl-0